news/2018-10-10-beatmap-spotlights-summer-2018.md

```diff
@@ -0,0 +1,309 @@
+---
+layout: post
+title: "Beatmap Spotlights Summer 2018"
+date: 2018-10-10 06:00:00 +0000
+---
+
+Welcome back to a long-awaited rendition of this season's beatmap spotlights! We have lots of brilliant maps to showcase and some extra surprise for all of you!
+
+![](/wiki/shared/news/2018-10-10-beatmap-spotlights-summer-2018/spotlight-header.jpg)
+
+The beatmap charts for the Summer Spotlights can be found [here](https://osu.ppy.sh/p/chart?ch=SUMMER2018). Their respective packs are available at the [Beatmap Pack Listing](https://osu.ppy.sh/p/packlist?t=r) for easier access. **The Summer 2018 Spotlights rankings will end on 31th of December, 2018.**
+
+## Changes within the Spotlights!
+
+Without further ado let's get into some recent changes brought about to the spotlights program.
+
+You may have noticed that the release of spotlights have been somewhat inconsistent, but worry no longer! Spotlights will now be released 4 times per year, once per season, and organized as follows:
+
++ **Spring Spotlights**: Showcase maps ranked from March 1st to May 31st, and released on June 30th
++ **Summer Spotlights**: Showcase maps ranked from June 1st to August 31th, and released on September 30th
++ **Fall Spotlights**: Showcase maps ranked from September 1st to November 30th, and released on December 31st
++ **Winter Spotlights**: Showcase maps ranked from December 1st to February 28th or 29th, and released on March 31st.
+
+If you've ever been interested in joining the team itself, open applications will now be made public through Google forms! The size of the spotlights staff is continuously growing, with structured teams for all 4 game modes alike. If you think you've got the mapping experience and skills to showcase truly outstanding maps, you can find the application [here](https://goo.gl/XdNvXo)! If you just want to observe, we decided to make our discord server public and you are [free to join](http://discord.gg/Z28ZtUf)! With that said, discussion about the future and other important stuff will still happen on the dedicated channel of [osu!dev](Http://discord.gg/ppy).
+
+We're also looking to integrate spotlight further into gameplay to create a more competitive experience. The top 3 players who manage to conquer the spotlights leaderboards can look forward to receiving 3, 2, and 1 months of supporter for 1st, 2nd, and 3rd place respectively. We're also looking to give temporary **grandmaster**, **expert**, and **pro** titles to anyone who can manage to prove their strength.
```

Comment:
capitalise titles?

Reply:
Titles are removed based on the "discussion" with Ephemeral